### PR TITLE
refactor: update `stats/base/dnanminabs` native addon from C++ to C

### PR DESCRIPTION
**Pull request by vivekmaurya001**
Resolves None.

## Description

- adds C addons to stats/base/dnanminabs instead of c++ addons

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

 - [x] Read, understood, and followed the [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md)

* * *

@stdlib-js/reviewers 

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+static napi_value addon( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 3 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
+	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 2 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, stride, argv, 1 );
+	STDLIB_NAPI_CREATE_DOUBLE( env, stdlib_strided_dnanminabs( N, X, stride ), v );
+	return v;
+}
+
+STDLIB_NAPI_MODULE_EXPORT_FCN( addon )
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c]: Missing trailing newline.

File: lib/node_modules/@stdlib/stats/base/dnanminabs/manifest.json [hunk 2]
```diff
@@ -24,6 +26,46 @@
...
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lm"
+      ],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/abs",
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dnanminabs/manifest.json]: Why is this dependency needed, if it wasn't previously? Same question on L41.

File: lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c (hunk 1 above)
Comment (vivekmaurya001) [reply lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c]: yy , sorry i forgot to add this

File: lib/node_modules/@stdlib/stats/base/dnanminabs/manifest.json (hunk 2 above)
Comment (vivekmaurya001) [reply lib/node_modules/@stdlib/stats/base/dnanminabs/manifest.json]: i have mistakenly added this , removed it now

File: lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c (hunk 1 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dnanminabs/src/addon.c]: If you have properly setup EditorConfig in your IDE, this should be automatically taken care of for you.

**Review by kgryte — APPROVED** [review]
LGTM.

**Pull request merged**